### DISCUSSION TITLE
电脑端导入Clash，无法使用

### DISCUSSION THREAD

**Discussion by liu-ziting** [vvbbnn00/WARP-Clash-API#86] [Q&A] [answered]
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/57311725/63663cc5-c5f5-422c-9145-29aca6cf28bb)

**Comment by vvbbnn00** [comment, marked as answer]
你好，请使用支持wireguard协议的clash内核，例如clash meta，例如支持该内核的桌面客户端：https://github.com/clash-verge-rev/clash-verge-rev

Hello, please use a Clash kernel that supports the WireGuard protocol, such as Clash Meta. For example, a desktop client that supports this kernel can be found at: https://github.com/clash-verge-rev/clash-verge-rev
